flatisfy/config.py

```diff
@@ -136,6 +137,10 @@ def _check_constraints_bounds(bounds):
             assert "type" in constraint
             assert isinstance(constraint["type"], str)
             assert constraint["type"].upper() in ["RENT", "SALE", "SHARING"]
+            
+            assert "minimum_pictures" in constraint
+            assert isinstance(constraint["minimum_pictures"], int)
```

Comment:
Could you consider that `None` is a valid value (and is the same as specifying 0)?

Reply:
Also, could you rename `minimum_pictures` to `minimum_photos`? This would make the spelling homogeneous across the app (so that we always use the term "photo" for photos, both in the code and the UI). Thanks!